### ISSUE TITLE
Various errors.

### ISSUE THREAD

**Issue by KateReller** [RiiConnect24/RiiConnect24-Patcher#130]
First error, happens when any sound tries to play:

```
New-Object : A positional parameter cannot be found that accepts argument
'Spencer\AppData\Roaming\RiiConnect24Patcher\sounds\warning3.wav'.
At line:1 char:2
\+ (New-Object System.Media.SoundPlayer C:\Users\Mary Spencer\AppData\Ro ...
\+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    \+ CategoryInfo          : InvalidArgument: (:) [New-Object], ParameterBindingException
    \+ FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.NewObjectCommand
```

Second error, happens after checking for updates: 

```
curl: (23) Failure writing output to destination
curl (6) Could not resolve host: Spencer\AppData\Roaming\RiiConnect24Patcher\internet\temp\announcement.txt
```

Third error, happens when going through the patching process for Dolphin:
```
ERROR.
There was an error while patching.
Error Code: 22
Failing module: Downloading Japan Delta / 12
```

**Comment by KcrPL** [comment]
About the third error, it has been fixed - we had some issues with our hosting.

About the first and second error, it will be fixed in the next release of the patcher. These are non-critical so nothing to worry about, you may continue.

**Comment by KcrPL** [comment]
Fixed in c0ffafbf1fa829ec6a79417f8b177155ea2aaf40
